docs/user-guide/troubleshooting.md

```diff
@@ -1,3 +1,11 @@
+<!--
+-----------------NOTICE------------------------
+This file is referenced in code as
+https://github.com/kubernetes/dashboard/blob/master/docs/user-guide/troubleshooting.md
+Do not move it without providing redirects.
```

Comment:
I think the project wiki is intended for these usecases, but not really sure how to set that up nicely